modules/core/src/main/java/org/apache/ignite/internal/processors/task/monitor/ComputeTaskStatusDiff.java

```diff
@@ -39,10 +39,11 @@
     boolean hasAttributesChanged();
 
     /**
-     * Only {@link ComputeTaskStatusSnapshot#status} has changed, if it is equal to
-     *      {@link ComputeTaskStatusEnum#FAILED}, then you can get {@link ComputeTaskStatusSnapshot#failReason}.
+     * {@link ComputeTaskStatusSnapshot#status} and {@link ComputeTaskStatusSnapshot#endTime} have changed,
+     *      if it is equal to {@link ComputeTaskStatusEnum#FAILED}, then you can get
```

Comment:
What's up with the padding? Do we need it?

Reply:
Of course it is not necessary, these are not arguments.

Reply:
fix it